### ISSUE TITLE
Flash plugin no longer working

### ISSUE THREAD

**Issue by Niek** [berstend/puppeteer-extra#131]
Moved [this comment](https://github.com/berstend/puppeteer-extra/issues/75#issuecomment-566066359) to a new issue for visibility:

Flash support is (no longer) working in at least Chrome 80, sample code:
```javascript
'use strict'

const puppeteer = require('puppeteer-extra')

const userName = require('os').userInfo().username
const pluginVersion = '32.0.0.303'
const pluginPath = `
  /Users/${userName}/Library/Application Support/Google/Chrome/PepperFlash/${pluginVersion}/PepperFlashPlayer.plugin
`.trim()

puppeteer.use(
    require('puppeteer-extra-plugin-flash')({
        pluginPath, // this should not be necessary, but setting just in case
        pluginVersion // the result is the same with or without
    })
);

(async () => {
    const browser = await puppeteer.launch({ headless: false, executablePath: '/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome', })
    const page = await browser.newPage()
    await page.goto('http://ultrasounds.com', { waitUntil: 'domcontentloaded' })
})()
```

Result: 
<img width="428" alt="Screen Shot 2019-12-16 at 14 37 13" src="https://user-images.githubusercontent.com/213140/70911271-d3e8be80-2011-11ea-87f7-b9e98fe84f0c.png">

chrome://version shows the correct Flash plugin/version, so I think it's more of a preference issue - this code no longer seems to work: https://github.com/berstend/puppeteer-extra/blob/491c82d042a8607e41d422b3e6bd8806292a151c/packages/puppeteer-extra-plugin-flash/index.js#L73-L110

**Comment by berstend** [comment]
Thanks for opening a new PR, I missed the comments on the old one.

Mhm, given how Flash is on it's way out Google is more and restricting it from being used (with the goal to eventually remove it), so this might be the reason the pptr extra plugin isn't working anymore.

I implemented this plugin 2 years and many major Chrome versions ago, when I had a use-case for it. :)

The way I found those preferences back then: Use a regular Chrome with a fresh profile and jump through all hoops to enable flash on a certain site, then diff the profile settings.

If there's still a need for this to work I can look into it, I also linked to an alternative using chrome management policies in the readme that might still work.

PS: Google will remove flash support altogether end of this year:
https://www.chromium.org/flash-roadmap#TOC-Flash-Support-Removed-from-Chromium-Target:-Chrome-87---Dec-2020-

**Comment by berstend** [comment]
My guess is that this change stopped it from working (timeline fits):

![image](https://user-images.githubusercontent.com/1368633/71933833-542eb980-31a3-11ea-82bb-d60059246966.png)

**Comment by Niek** [comment]
Found a fix, see PR https://github.com/berstend/puppeteer-extra/pull/133 - closing this.

**Comment by berstend** [comment]
Fix has been published as `puppeteer-extra-plugin-flash@2.2.5`

**Comment by githubyandou524** [comment]
> Fix has been published as `puppeteer-extra-plugin-flash@2.2.5`

I tried, but still not work, can you help me?
